client/packages/invoices/src/Returns/modals/InboundReturn/InboundReturn.tsx

```diff
@@ -60,12 +69,13 @@ export const InboundReturnEditModal = ({
   });
 
   useEffect(() => {
-    if (initialItemId) setItemId(initialItemId);
+    if (initialItemId === undefined) return;
+    setItemId(initialItemId === null ? undefined : initialItemId);
   }, [initialItemId]);
 
   const onOk = async () => {
     try {
-      await save();
+      !isDisabled && (await save());
```

Comment:
As mentioned in your PR, this will just close the modal without attempting to save when disabled.